### DISCUSSION TITLE
Can we add the "Reveal in File Explorer" features?

### DISCUSSION THREAD

**Discussion by ouliuquan** [eclipse-theia/theia#12072] [Ideas] [answered]
For now Theia don't have `Reveal in File Explorer`, Theia's `Reveal in Explorer` only locate file in resouce manager.
I think this real functional, can we consider adding this feature?

![open_in_explorer](https://user-images.githubusercontent.com/38776890/212586125-586cfccf-2d94-4c52-9b6b-1ffddaebc9e3.png)

**Comment by msujew** [comment, marked as answer]
@ouliuquan To keep things actionable, we create feature proposals as issues. I've created https://github.com/eclipse-theia/theia/issues/12073 for you.

**Reply by ouliuquan** [reply]
@msujew Thanks Mark, it's very nice of you, I feel happy to see this issue.
